server/views/applications/pages/check-your-answers/check-your-answers.njk

```diff
@@ -44,13 +49,32 @@
         {% endfor %}
       {% endfor %}
 
-      <form action="{{ paths.applications.pages.update({ id: applicationId, task: task, page: page.name }) }}?_method=PUT" method="post">
+      <form action="{{ paths.applications.pages.update({ id: applicationId, task: task, page: page.name }) }}?_method=PUT" method="post" class="govuk-!-margin-top-8">
         <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
 
-        <input type="hidden" name="reviewed" value="1"/>
+        {{
+          formPageCheckboxes(
+            {
+              fieldName: "checkYourAnswers",
+              fieldset: {
+                legend: {
+                  text: "Confirm your answers",
+                  classes: "govuk-fieldset__legend--l"
+                }
+              },
+              items: [
+                {
+                  value: "reviewed",
```

Comment:
Very minor, but I wonder if "confirmed" might be a more accurate term here? I think that's what we're really seeking here? Confirmation that the answers are correct, rather than affirmation that the answers have been reviewed?